contrib/devtools/security-check.py

```diff
@@ -152,6 +152,24 @@ def check_PE_Canary(binary) -> bool:
     '''
     return binary.has_symbol('__stack_chk_fail')
 
+def check_PE_SecureZeroMemory(binary) -> bool:
+    '''
+    Check for use of SecureZeroMemory()
+    '''
+    cleanse = binary.get_symbol('_Z14memory_cleansePvy').value
+
+    section_addr = binary.section_from_rva(cleanse).virtual_address
+    virtual_address = binary.optional_header.imagebase + section_addr + cleanse
+
+    content = binary.get_content_from_virtual_address(virtual_address, 40, lief.Binary.VA_TYPES.VA)
+
+    # We are looking for rep stosb, which is f3 aa (243 170).
+    # We search for 170, and check for a preceding 243,
+    # so we don't match the endbr64 instruction at the
+    # beginning of the function.
+    aa = content.index(170)
+    return content[aa-1] == 243
```

Comment:
This checks that the function is there, not whether it is used, in the right places. But it may be enough as a start.

Reply:
> Oh never mind, you're not looking for SecureZeroMemory itself, only its implementation within memory_cleanse. This makes sense.

Yea. Happy to try and extend / improve things further where we can.